monkey/monkey_island/cc/ui/src/components/report-components/security/AvailableCredentials.tsx

```diff
@@ -0,0 +1,31 @@
+import {getAllUsernames, getAllSecrets} from '../credentialParsing';
+import React from 'react';
+
+const AvailableCredentials = (props) => {
```

Comment:
These are available, not used. This will show up even if no brute-force exploiters were enabled